### PR TITLE
Add the ability to add volumes to master/worker pods 

### PR DESCRIPTION
**Pull request by nireasybizy**
could be useful if someone wants to add their own locust file

### CODE REVIEW

File: helm/locust/templates/deployment-master.yaml [hunk 1]
```diff
@@ -57,6 +57,10 @@ spec:
           - name: {{ $key }}
             value: {{ $value | quote }}
           {{- end }}
+          volumeMounts:
+          {{- if .Values.master.extraVolumeMounts }}
+            {{- toYaml .Values.master.extraVolumeMounts | nindent 12 }}
+          {{- end }}
```
Comment (mosheavni) [inline helm/locust/templates/deployment-master.yaml]:
```suggestion
          {{- if .Values.master.volumeMounts }}
          volumeMounts:
            {{- toYaml .Values.master.volumeMounts | nindent 12 }}
          {{- end }}
```

File: helm/locust/templates/deployment-master.yaml [hunk 2]
```diff
@@ -69,3 +73,7 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+      volumes:
+      {{- if .Values.master.extraVolumes }}
+        {{ toYaml .Values.master.extraVolumes | nindent 8 }}
+      {{- end }}
```
Comment (mosheavni) [inline helm/locust/templates/deployment-master.yaml]:
```suggestion
      {{- if .Values.master.volumes }}
      volumes:
        {{ toYaml .Values.master.volumes | nindent 8 }}
      {{- end }}
```

File: helm/locust/templates/deployment-worker.yaml [hunk 3]
```diff
@@ -50,6 +50,10 @@ spec:
           - name: {{ $key }}
             value: {{ $value | quote }}
           {{- end }}
+          volumeMounts:
+          {{- if .Values.worker.extraVolumeMounts }}
+            {{- toYaml .Values.worker.extraVolumeMounts | nindent 12 }}
+          {{- end }}
```
Comment (mosheavni) [inline helm/locust/templates/deployment-worker.yaml]:
```suggestion
          {{- if .Values.worker.volumeMounts }}
          volumeMounts:
            {{- toYaml .Values.worker.volumeMounts | nindent 12 }}
          {{- end }}
```

File: helm/locust/templates/deployment-worker.yaml [hunk 4]
```diff
@@ -62,3 +66,7 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+      volumes:
+      {{- if .Values.worker.extraVolumes }}
+        {{ toYaml .Values.worker.extraVolumes | nindent 8 }}
+      {{- end }}
```
Comment (mosheavni) [inline helm/locust/templates/deployment-worker.yaml]:
```suggestion
      {{- if .Values.worker.volumes }}
      volumes:
        {{ toYaml .Values.worker.volumes | nindent 8 }}
      {{- end }}
```

File: helm/locust/values.yaml [hunk 5]
```diff
@@ -36,6 +36,15 @@ master:
     minReplicas: 1
     maxReplicas: 1
     targetCPUUtilizationPercentage: 50
+  extraVolumeMounts: []
+  ## Additional volumeMounts to the master container
+  #  - name: locust-file
+  #   mountPath: /locust
+  extraVolumes: []
+  ## Additional volumes to the master pod.
```
Comment (mosheavni) [inline helm/locust/values.yaml]:
```suggestion
  volumeMounts: []
  ## volumeMounts for the master container
  #  - name: locust-file
  #   mountPath: /locust
  volumes: []
  ## volumes for the master pod.
```

File: helm/locust/values.yaml [hunk 6]
```diff
@@ -53,6 +62,15 @@ worker:
...
```
Comment (mosheavni) [inline helm/locust/values.yaml]:
```suggestion
  volumeMounts: []
  ## volumeMounts to the master container
  #  - name: locust-file
  #   mountPath: /locust
  volumes: []
  ## volumes for the master pod.
```

**Review by mosheavni — CHANGES_REQUESTED** [review]

**Review by hansehe — APPROVED** [review]

**Comment by hansehe** [comment]
Thanks! I'll merge it and push a new chart version

**Pull request merged**